yarn.lock

```diff
@@ -5799,6 +5799,11 @@ json5@^2.1.2, json5@^2.2.0:
   dependencies:
     minimist "^1.2.5"
 
+jsonc-parser@^3.0.0:
```

Comment:
Does Monaco depend on some jsonc parser already that we could use?

Reply:
It looks like Monaco depends on the same jsonc-parser package: https://github.com/microsoft/monaco-editor/blob/7826dee4ea80e7e4542db6934593e370f2305d97/package.json#L43. I think the jsonc-parser interface is a lot more straightforward than the Monaco interface, but it might be possible to wrap Monaco.

Reply:
It's a dev dependency in Monaco. 

I'm asking since I would have expected the dependency to already be in the lock file.